app/src/main/java/org/piramalswasthya/sakhi/work/UpdatePNCToECWorker.kt

```
@@ -44,29 +45,43 @@ class UpdatePNCToECWorker @AssistedInject constructor(
+                    it.updatedDate = now
+                    it.genDetails?.reproductiveStatusId = 1
+                    it.genDetails?.reproductiveStatus =
+                        applicationContext.resources.getStringArray(R.array.nbr_reproductive_status_array)[0]
+                    if (it.processed != "N") it.processed = "U"
+                    it.syncState = SyncState.UNSYNCED
+                }
+                if (ben != null) {
+                    benRepo.updateRecord(ben)
+                }
             }
+        }catch (e:Exception){
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)